__tests__/app.test.js

```diff
@@ -124,4 +124,51 @@ describe('GET /api/articles', () => {
           })
       })
   });
+});
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with array of comments for given article id with most recent comments first', () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            const {comments} = body;
+            expect(comments).toHaveLength(11);
```

Comment:
Great job remembering to check that the array is not empty when asserting inside a forEach